### ISSUE TITLE
Typography - text color

### ISSUE THREAD

**Issue by xinyil** [Princeton-CDH/derrida-django#155]
I've noticed this for a long time and it might benefit from putting it in a separate ticket:
A lot of the text is set in a dark grey, while they should be black. We never had a dark grey on the site, some lighter grey is used for inactive elements. All major text is black and the contrast is important.

Please check.

**Comment by kevinglover** [comment]
@xinyil I was worried about high contrast such as black on white because it can cause eye strain. Also there is evidence that it is particularly bad for people with dyslexia http://uxmovement.com/content/6-surprising-bad-practices-that-hurt-dyslexic-users/

The Typography Handbook, a reference guide to web design also warns against too much contrast, and recommends developers build using a very dark gray (#333) instead of pitch black (#000).
http://typographyhandbook.com/#color

**Comment by xinyil** [comment]
You're right grey is soft and I'm aware of some discussion of this issue. I was speaking of the importance of black from an art direction stand point of view - we wanted sharp contrast instead of muted techy feel, and we already used off-white background.

**Comment by rlskoeser** [comment]
@kevinglover I appreciate your concern about eye strain and dyslexia - thanks for sharing the references.  This is something we should have discussed before you made changes, though.

@kevinglover I see you've committed style changes - is this complete?

**Comment by thatbudakguy** [comment]
think I've finished what kevin started.

**Comment by xinyil** [comment]
@thatbudakguy when navigation text is not selected, it's still grey. hovering and selected ones turn black.

**Comment by thatbudakguy** [comment]
fixed in #76 .

**Comment by xinyil** [comment]
in L viewport, I'm still experiencing a grey to black shift on hover

**Comment by thatbudakguy** [comment]
can't duplicate. what browser? can you post a video?

**Comment by thatbudakguy** [comment]
Need to make all the grays darker, as per accessibility review. Maybe WAVE can tell us how dark to make them.

**Comment by xinyil** [comment]
@thatbudakguy I've updated the greys and tagged you in the "citation" screen. I've checked the contrast with a sketch plug-in.

**Comment by thatbudakguy** [comment]
took at stab at this. @xinyil let me know how I did, I'm sure I missed something

**Comment by xinyil** [comment]
It looks good with bare eyes. We should run WAVE and see the results?
A few things:

- can we keep the divider lines and input boxes outline #D8D8D8? they look too stark now.
- all underlines should be #D8D8D8. after this update, it seems that the "clear all" is missing its underline and the underline of permalink turned white.
<img width="1245" alt="screen shot 2018-04-30 at 10 11 28 am" src="https://user-images.githubusercontent.com/17734456/39431509-f46e2b0a-4c5e-11e8-88a0-a43ba21405bb.png">

<img width="923" alt="screen shot 2018-04-30 at 10 01 38 am" src="https://user-images.githubusercontent.com/17734456/39431473-d5803d78-4c5e-11e8-9c0a-c3b7e0934b37.png">

**Comment by rlskoeser** [comment]
The permalink underline color is fixed.  I think we can live with the other items for current release.

**Comment by xinyil** [comment]
@rlskoeser  the underline of permalink and clear button still appear to be white on test site?

**Comment by rlskoeser** [comment]
@xinyil you're right! I thought I saw the permalink underline got fixed, but I must have confused which site I was looking at.

**Comment by rlskoeser** [comment]
Gallery permalink default underline color is now fixed, however:
- vertical placement seems off: when I select one, the gray underline is showing below the red
- hover color should be red and not blue 

older issues:
- gallery divider line `.gallery-image-view__list-end` should be #D8D8D8, currently #737373
- unselected corner nav is #424242, not true black; `.mdl-navigation__link`

**Comment by rlskoeser** [comment]
Just reviewed, everything I identified in the last round is now correct.
